test/test_common/real_threads_test_helper.h

```diff
@@ -1,3 +1,5 @@
+#pragma once
+
 #include "source/common/event/dispatcher_impl.h"
 #include "source/common/thread_local/thread_local_impl.h"
 
```

Comment:
can you add a class comment that ReallThreadsTestHelper is used both for performance and unit tests, and thus no mocks or other slow test structures should be added to it.